sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/SourceOptions.scala

```diff
@@ -47,4 +47,6 @@ object SourceOptions {
   val RESPECT_SPARK_SCHEMA = "respectSparkSchema"
   val DEFAULT_RESPECT_SPARK_SCHEMA = false
 
```

Comment:
maybe all the `XXXOptions` should extend `SourceOptions`, so that we can put common code here, e.g.
```
val ignoreCorruptFiles: Boolean = parameters.get("ignoreCorruptFiles").map(_.toBoolean)
    .getOrElse(SQLConf.get.ignoreCorruptFiles)
```